api/v1/kataconfig_types.go

```diff
@@ -59,8 +59,6 @@ type KataConfigStatus struct {
 	// Upgradestatus reflects the status of the ongoing kata upgrade
 	// +optional
 	Upgradestatus KataUpgradeStatus `json:"upgradeStatus,omitempty"`
-
-	BaseMcpGeneration int64 `json:"prevMcpGeneration"`
```

Comment:
This is an API change. Will this fly if the server has an old KataConfig with the generation field ?

Reply:
The controller should be fine since it just never references the field anymore.  The control plane ignores unknown fields by default, and AFAIK  it only does this checking on committing resource creation & update.  From our previous experience it appears that when the updated controller checks a KataConfig instance back into database unknown fields (wrt to the new CRD schema) are dropped (we had this situation e.g. with the `.spec.kataMonitorImage` already).